#!/usr/bin/env python
PACKAGE = "dvs_renderer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("frame_rate", int_t, 0, "Frame rate (Hz)", 60, 1, 200)
gen.add("synchronize_on_frames",   bool_t,   0, "Whether to synchronize the event rendering generation on the APS frames",  False)
gen.add("use_color",   bool_t,   0, "Color (True) or grayscale (False)",  True)
gen.add("use_fixed_num_events",   bool_t,   0, "Fixed num events (True) or milliseconds (False)",  True)
gen.add("frame_size",    int_t,    0, "Frame size", 5000,  0, 1000000)

exit(gen.generate(PACKAGE, "dvs_renderer", "DVS_Renderer"))
